test/jdk/sun/net/www/http/KeepAliveCache/B5045306.java

```diff
@@ -133,6 +139,9 @@ public static void clientHttpCalls() {
         } finally {
             server.terminate();
         }
+        if (!uncaught.isEmpty()) {
+            throw new AssertionError("Unhandled exception:", uncaught.get(0));
```

Comment:
The existing test throws RuntimeException when it fails, maybe this case should do the same.